yt/visualization/volume_rendering/lens.py

```diff
@@ -14,6 +14,8 @@
 # ----------------------------------------------------------------------------
 
 from __future__ import division
+from astropy.coordinates import spherical_to_cartesian
+from astropy_healpix import HEALPix
```

Comment:
These astropy imports can't happen at the top level since astropy is an optional dependency of yt. Instead, I'd suggest writing wrappers for the modules you need using the on-demand import infrastructure. See `yt/utilities/on_demand_imports.py`.

Reply:
Thanks! I'll change this.

Reply:
Should I make astropy_healpix part of astropy in on_demand_imports? They are two different packages but they are closely related.

Reply:
It’s a separate module so it should get its own on demand import wrapper.